collect_app/src/test/java/org/odk/collect/android/activities/FormMapViewModelTest.java

```diff
@@ -31,87 +31,87 @@
     }
 
     @Test public void getFormTitle_returnsFormTitle() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm1, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_1, testInstancesRepository);
         assertThat(viewModel.getFormTitle(), is("Form with ID 1"));
     }
 
     @Test public void getFormId_returnsFormId() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm1, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_1, testInstancesRepository);
         assertThat(viewModel.getFormId(), is("formId1"));
     }
 
     @Test public void getTotalInstanceCount_returnsCountOfAllInstances() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm1, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_1, testInstancesRepository);
         assertThat(viewModel.getTotalInstanceCount(), is(7));
     }
 
     @Test public void getMappableInstances_excludesInstancesWithoutGeometry() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm1, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_1, testInstancesRepository);
         List<FormMapViewModel.MappableFormInstance> mappableFormInstances = viewModel.getMappableFormInstances();
 
         assertThat(mappableFormInstances.size(), is(6));
         assertThat(mappableFormInstances, not(contains(hasProperty("databaseId"), is(3L))));
     }
 
     @Test public void getMappableInstances_excludesInstancesWithCorruptGeometry() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm2, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_2, testInstancesRepository);
         List<FormMapViewModel.MappableFormInstance> mappableFormInstances = viewModel.getMappableFormInstances();
 
         assertThat(mappableFormInstances.size(), is(0));
     }
 
     @Test public void getDeletedDateOf_returnsDeletedDate() {
-        FormMapViewModel viewModel = new FormMapViewModel(testForm1, testInstancesRepository);
+        FormMapViewModel viewModel = new FormMapViewModel(TEST_FORM_1, testInstancesRepository);
 
         assertThat(viewModel.getDeletedDateOf(0L), is(1487782554846L));
```

Comment:
Minor nitpick: could the assertion be that it's equal to `testInstance[0].getDeletedDate()` instead. Just a bit hard to track where this value is coming from.

Reply:
Thanks, done.